packages/core/src/__tests__/ceramic.test.ts

```diff
@@ -52,6 +55,8 @@ const createCeramic = async (ipfs: IpfsApi, anchorOnRequest = false): Promise<Ce
 }
 
 const anchor = async (ceramic: Ceramic): Promise<void> => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
The `anchor` method doesn't exist in the `AnchorService` interface in this case.

Reply:
Can we add it then?

Reply:
No, we can't do that since `anchor` is not part of the `AnchorService` API. The method anchors immediately and exists in the in-memory anchor service only

Reply:
Got it, that makes sense.  Thanks for explaining!